src/cli/README_HISTORY.md

```diff
@@ -0,0 +1,290 @@
+# OpenThread CLI - History Tracker
+
+History Tracker module records history of different events (e.g., RX and TX IPv6 messages or network info changes, etc.) as the Thread network operates. All tracked entries are timestamped.
+
+All commands under `history` require `OPENTHREAD_CONFIG_HISTORY_TRACKER_ENABLE` feature to be enabled.
+
+The number of entries recorded for each history list is configurable through a set of OpenThread config options, e.g. `OPENTHREAD_CONFIG_HISTORY_TRACKER_NET_INFO_LIST_SIZE` specifies the number of entries in Network Info history list. The History Tracker will keep the most recent entries overwriting oldest one when the list gets full.
+
+## Command List
+
+Usage : `history [command] ...`
+
+- [help](#help)
+- [netinfo](#netinfo)
+- [rx](#rx)
+- [tx](#tx)
+
+## Timestamp Format
+
+Recorded entries are timestamped. When the history list is printed, the timestamps are shown relative the time the command was issues (i.e., when the list was printed) indicating how long ago the entry was recorded.
+
+```bash
+> history netinfo
+| Time                     | Role     | Mode | RLOC16 | Partition ID |
```

Comment:
How about renaming column to "Age" so we do not have to print "ago" in each entry?